build.sbt

```diff
@@ -11,7 +11,11 @@ ThisBuild / mergifyStewardConfig ~= {
 
 ThisBuild / crossScalaVersions := Seq("2.13.12", "3.3.1")
 
-lazy val root = tlCrossRootProject.aggregate(toolkit, toolkitTest)
+lazy val root = tlCrossRootProject
+  .aggregate(toolkit, toolkitTest)
+  .settings(
+    (Test / test) := (Test / test).dependsOn(toolkitTesting / Test / test).value
+  )
```

Comment:
Oh, why did you add this? This is why its being called in the JS/Native jobs.

Instead, you should add `toolkitTesting` to the aggregate.

Reply:
hmm, you might be right

Reply:
it fixed!